website/src/components/Explain.tsx

```diff
@@ -11,7 +11,8 @@ import {
 import { Info } from "lucide-react";
 
 interface ExplainProps {
-  explanation: string[];
+  // Each element is an explanation for individual item (e.g. flag)
+  explanation: Array<typeof Text>;
```

Comment:
Maybe replace this with ReactElement[] would work

Reply:
Yeah, i think this will fix the build issue.